lib/house.rb

```diff
@@ -0,0 +1,42 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @above_market_average = false
+    @sum = 0
+    @area_room = []
+    @detail_hsh = {}
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    @above_market_average
```

Comment:
since you set it to a default value of false in `def initialize`, there's no way a house can ever be above market average.  This should be `@price > 500000` instead (if the price is greater than 500000, this statement will return true, or else, it will return false.